mlperf.conf

```diff
@@ -28,28 +30,33 @@ stable-diffusion-xl.*.performance_sample_count_override = 5000
 
 *.SingleStream.target_latency_percentile = 90
 *.SingleStream.min_duration = 600000
-#*.SingleStream.min_query_count = 1024
 
 *.MultiStream.target_latency_percentile = 99
 *.MultiStream.samples_per_query = 8
 *.MultiStream.min_duration = 600000
-#*.MultiStream.min_query_count = 270336
 *.MultiStream.min_query_count = 662
 retinanet.MultiStream.target_latency = 528
 
+# With early stopping enabled, the min_query_count is 1 for Server Scenario
+*.Server.min_query_count = 1
```

Comment:
IIRC this is also 100. Need to confirm though.

Reply:
Hmm if min_query_count for server is 100, equal_issue_mode will be very burdensome to run. Let's discuss tomorrow.

Reply:
yes, that's correct. Even otherwise, `equal_issue_mode` is effectively making the server scenario behave as offline scenario right?

Reply:
I think the implementation in Server is still incorrect. It shouldn't run like Offline (e.g. 24576 as sample_per_query). I will fix it and test

Reply:
I have changed the behavior, so sample_per_query is still 1 in server the singlestream.